.gitignore

```diff
@@ -1 +1,8 @@
 isopod*
+
+# IDEs
+.idea
+.vscode
+
+# local directories
+local
```

Comment:
add new line
pro tip: configure your editor to auto insert new line at end of file such as https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files